build.sbt

```diff
@@ -72,20 +68,12 @@ lazy val bsp4j = project
     crossVersion := CrossVersion.disabled,
     autoScalaLibrary := false,
     Compile / javacOptions ++= {
-      val specifyRelease =
-        if (sys.props("java.version").startsWith("1.8"))
-          List.empty
-        else
-          List("--release", "8")
```

Comment:
Why do we want to ensure it? Java 8 is EOL anyway

Reply:
> Why do we want to ensure it? Java 8 is EOL anyway

Because a massive amount of slow-moving companies and projects are still on it. So if it's not a burden to maintain, that I'd recommend possibly keeping it. If truly none of the tools listed want it, then sure let's target 11, but I'm not sure that's the case.

CC @lefou

Reply:
Okay, let's just drop it for now

Reply:
I meant dropping the change, not java 8, sorry! I already reverted it in the newest revision :)